app/views/shared/_dap_analytics.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= t('notices.dap_html') %>
+<% dap_source = 'https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA' %>
+<%= nonced_javascript_tag({src: dap_source, async: true, id: '_fed_an_ua_tag'}) do %>
```

Comment:
My ruby / rails is a little rusty so maybe there is a more appropriate way to use the library...but this seems to resolve the issue